terraform/environments/core-logging/route53_resolver_logs.tf

```diff
@@ -1,8 +1,18 @@
+provider "aws" {
+  alias  = "us-east-1"
+  region = "us-east-1"
+}
+
 resource "aws_cloudwatch_log_group" "modernisation-platform-r53-resolver-logs" {
   name              = "modernisation-platform-r53-resolver-logs"
   retention_in_days = 365
 }
 
+provider "aws" {
```

Comment:
This provider is not used it can be removed

Reply:
Done and re-pushed